### DISCUSSION TITLE
Is there a version that workes with inkscape .92 ?

### DISCUSSION THREAD

**Discussion by oldunixguy** [inkstitch/inkstitch#1924] [General]
For reasons that are too involved to go into here, I have inkscape .92 on an older version of Ubuntu.
Neither can be upgraded.
Is there a version of inkstitch that I can download to run with these? If so, where do I find it?
thanks
oldunixguy

**Comment by kaalleen** [comment]
I suppose that would be this version here: https://inkstitch.org/en/2020-08-19-v1.29.1

**Comment by oldunixguy** [comment]
Great! BUT.... when I scroll down the page to the Assets section all that appears is a rotating spinner and not a single file list. It spins forever! Any idea on how to cure this?
thanks
oldunixguy

**Comment by oldunixguy** [comment]
I selected a screen shot image to be attached to the previous post but it does not show up. What is happening with this github web page? I never used to have problems accessing github pages or using the web page interface to download .zip, etc.
help!
thanks
oldunixguy

**Comment by kaalleen** [comment]
For me everything looks fine? Maybe you just have to try again.
